vs-boost-unit-test-adapter/TSAOptions.json

```diff
@@ -0,0 +1,21 @@
+{
+    "tsaVersion": "TsaV2",
+    "codebase": "NewOrUpdate",
+    "codebaseName": "VCLS BoostTest GitHub",
+    "tsaStamp": "DevDiv",
+    "tsaEnvironment": "PROD",
+    "notificationAliases": [
+       "davidraygoza@microsoft.com",
+       "cpp-apogee@microsoft.com"
```

Comment:
can you use the new `cppintellisense` alias instead? Same for line 13.

Reply:
Will do in future PR, have to update Google test TSA options as well